src/align.h

```diff
@@ -4,8 +4,8 @@
 #define T_SIZE 32           // Size of a tile in the bayer mosaiced image
 #define T_SIZE_2 16         // Half of T_SIZE and the size of a tile throughout the alignment pyramid
 
-#define MIN_OFFSET -168     // Min total alignment (based on three levels and downsampling by 4)
-#define MAX_OFFSET 126      // Max total alignment. Differs from MIN_OFFSET because total search range is 8 for better vectorization
+#define MIN_OFFSET -256     // Min total alignment (based on three levels and downsampling by 4)
+#define MAX_OFFSET 256      // Max total alignment. Differs from MIN_OFFSET because total search range is 8 for better vectorization

```

Comment:
What are the performance impacts of this change? The search region is kept somewhat small such that it vectorizes cleanly. Does the processing run at a similar speed?

Reply:
i dont think it has a big performance impact. at least i dont feel it. but time is relative when you sit hours infront of it and diff images ;) I will trace the time it takes to process, bevor and after.
What i have noticed is that good tiles get sorted out when the camera got shaked to heavy and bad tiles that match that range gets more weighted wich can result in ghosting.